src/app.module.ts

```
@@ -8,11 +8,32 @@ import { MyLoggerModule } from './my-logger/my-logger.module';
+    StoreModule,
+    // Configuration de ClsModule au niveau racine
+    ClsModule.forRoot({
+      global: true, // Rend ClsModule globalement disponible
+      middleware: {
+        mount: true, // Monte le middleware CLS pour encapsuler chaque requête
+        // Options supplémentaires, par exemple pour générer un ID de requête unique
+        generateId: true,
+        idGenerator: (req: Request) =>
+          // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+          req.headers['x-request-id'] || 'unique-id-' + Math.random().toString(36).substring(2, 10),
+      },
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)